MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult+Internal.swift

```diff
@@ -26,25 +26,57 @@ import Foundation
 
 extension MSALNativeAuthUserAccountResult {
 
-    func getAccessTokenInternal(
-        forceRefresh: Bool,
-        correlationId: UUID?,
-        cacheAccessor: MSALNativeAuthCacheInterface
-    ) async -> MSALNativeAuthCredentialsControlling.RefreshTokenCredentialControllerResponse {
+    func getAccessTokenInternal(forceRefresh: Bool = false,
```

Comment:
we should not use default value in this method, but default value should be used in public methods

Reply:
Done.